### PR TITLE
YAML 1.2 and standarization of booleans

### PR DESCRIPTION
**Pull request by frenck**
**Description:**

Removes some last "truthy" boolean values from our documentation and also standardizes boolean usage to lower case.

Also: Fixed a couple of broken examples along the way.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/alert.markdown [hunk 1]
```diff
@@ -186,8 +186,8 @@ alert:
       - 15
       - 30
       - 60
-    can_acknowledge: True  # Optional, default is True
-    skip_first: True  # Optional, false is the default
+    can_acknowledge: true  # Optional, default is True
```
Comment (tjorim) [inline source/_components/alert.markdown]: Comment still mentions True

File: source/_components/device_tracker.markdown [hunk 2]
```diff
@@ -84,8 +84,8 @@ devicename:
...
 | `icon`         | mdi:account                   | An icon for this device (use as an alternative to `picture`).                           |
 | `gravatar`     | None                          | An email address for the device's owner. If provided, it will override `picture`.                        |
-| `track`        | [uses platform setting]       | If  `yes`/`on`/`true` then the device will be tracked. Otherwise its location and state will not update. |
-| `hide_if_away` | False                         | If `yes`/`on`/`true` then the device will be hidden if it is not at home.                                |
+| `track`        | [uses platform setting]       | If `true` then the device will be tracked. Otherwise its location and state will not update. |
+| `hide_if_away` | False                         | If `true` then the device will be hidden if it is not at home.                                |
```
Comment (tjorim) [inline source/_components/device_tracker.markdown]: False -> false?

File: source/_components/alert.markdown (hunk 1 above)
Comment (frenck) [reply source/_components/alert.markdown]: Fixed, thanks, well spotted 👀

File: source/_components/device_tracker.markdown (hunk 2 above)
Comment (frenck) [reply source/_components/device_tracker.markdown]: Thanks! Fixed! 👍

**Comment by tjorim** [comment]
I'm still wondering if state attributes are lowercase booleans as well, currently don't have a component that has it set up to check. E.g. https://www.home-assistant.io/components/geo_location.nsw_rural_fire_service_feed/

**Comment by frenck** [comment]
@tjorim I will check that myself as well.

**Comment by frenck** [comment]
Lets close this 😉

**Pull request closed without merging**